test/jdk/java/awt/dnd/DnDAcceptanceTest/MyMouseDragGestureRecognizer.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* Due to 2 AWT bugs:
+ * 4117525  mouseDragged events on Win32 do not set BUTTON1_MASK correctly
```

Comment:
These bugs were fixed 20 years ago. Is this sub-class still necessary ?

Reply:
`MyMouseDragGestureRecognizer` was being used in DnDSource.java.

> Is this sub-class still necessary ?

Sub-class is not necessary since there is a better approach - [Toolkit.getDefaultToolkit().createDragGestureRecognizer()](https://docs.oracle.com/en/java/javase/23/docs/api/java.desktop/java/awt/dnd/MouseDragGestureRecognizer.html) 
I have update DnDSource to use it instead of `MyMouseDragGestureRecognizer` and deleted the subclass.

Reply:
@prrace Can you please re-review when you get a chance?